.github/workflows/tests_lkcpu_python.yml

```diff
@@ -168,6 +168,22 @@ jobs:
           fetch-tags: true
           path: main
 
+      - name: Switch to release build of Lightning
+        if: inputs.lightning-version == 'release'
+        run: |
+          cd main
+          git fetch --all
+          git checkout $(git branch -a --list "origin/v*rc*" | tail -1)
+
+      - name: Switch to stable build of Lightning
+        if: inputs.lightning-version == 'stable'
+        run: |
+          cd main
+          git fetch --tags --force
+          git checkout $(git tag | sort -V | tail -1)
+          git log -1 --format='%H'
+          git status
```

Comment:
Would you like to simply install the the stable version from PyPI?

Reply:
Yes, but there is `SKIP_COMPILATION=True python -m pip install . -vv` under, which would overwrite it. So either that, or we introduce some more logic down there.